1735Sprint/03-Sheik/demo/src/main/java/com/qualogy/demo/App.java

```diff
@@ -0,0 +1,13 @@
+package com.qualogy.demo;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+    public static void main( String[] args )
+    {
+        System.out.println( "Hello World!" );
+    }
+}
```

Comment:
do you like when people tell you that you have a different code style than style people got used to?
https://google.github.io/styleguide/javaguide.html